pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1043,23 +1043,25 @@ public PinotResourceManagerResponse deleteBrokerTenantFor(String tenantName) {
   /**
    * Schema APIs
    */
-  public void addSchema(Schema schema, boolean override) {
+  public void addSchema(Schema schema, boolean override)
```

Comment:
can you add a test case for this change?

Reply:
Added the test, and also refined the error code